Wrong IgxGridDetailsPipe import from tree grid barrel

## Description  
Build error 
![image](https://user-images.githubusercontent.com/11193764/70068690-24672180-15f9-11ea-897f-b2ee6239bdae.png)


 * igniteui-angular version: 9.0.0-beta.5
 * browser: any

## Steps to reproduce  

1. Build the product with external project e.g. Angular Samples [PR](https://github.com/IgniteUI/igniteui-angular-samples/pull/1622)

## Result  
Error: error TS100: Can't resolve all parameters for ɵfu in D:/Dev/igniteui-angular-samples/node_modules/igniteui-angular/igniteui-angular.d.ts: (?).
![image](https://user-images.githubusercontent.com/11193764/70067916-b9691b00-15f7-11ea-8310-e2490f7401e6.png)

--- a/projects/igniteui-angular/src/lib/grids/grid/grid.details.pipe.ts
+++ b/projects/igniteui-angular/src/lib/grids/grid/grid.details.pipe.ts
@@ -1,7 +1,7 @@
 import { PipeTransform, Pipe } from '@angular/core';
-import { IGroupByResult } from '../../data-operations/grouping-result.interface';
-import { GridBaseAPIService, IgxGridBaseDirective } from '../tree-grid';
 import { GridType } from '../common/grid.interface';
+import { IgxGridBaseDirective } from '../grid-base.directive';
+import { GridBaseAPIService } from '../api.service';
 import { IgxGridAPIService } from './grid-api.service';
 
 /** @hidden */

